README.md

```diff
@@ -100,6 +157,7 @@ $("button").on("click", function(event){
 Your Answer:
 ```js
 // write code here
+document.getElementsByTagName("button").this.style.color = 'red';
 ```
```

Comment:
Because `getElementsByTagName` returns multiple DOM elements, you would need to iterate through the resulting collection and apply event listeners individually. 

You would also need to use `.addEventListener` in place of `.on`

So something like...

```js
var buttons = document.getElementsByTagName("button")
for(var i = 0; i < buttons.length; i++){
  buttons[i].addEventListener("click", function(evt){
    evt.target.style.color = "red";
 })
};